importer/DkBygningDa.py

```diff
@@ -52,15 +54,15 @@ def set_inception(self):
 
     def __init__(self, db_row_dict, mapping, data_files, existing):
         Monument.__init__(self, db_row_dict, mapping, data_files, existing)
-        self.update_labels()
-        self.exists("da")
-        self.set_commonscat()
-        self.set_image("billede")
-        self.set_coords(("lat", "lon"))
-        self.set_adm_location()
+        # self.update_labels()
```

Comment:
I understand that commenting them out while testing. But for merging the code having these commented out doesn't make sense (assuming they have been implemented).